docs/source/upgrade-guide/index.md

```diff
@@ -28,7 +28,21 @@ It is usually better and quicker to move your items into new locations and copy
 ```
 ````
 
-(volto-upgrade-guide-18.x.x)=
+(volto-upgrade-guide-19.x.x)=
+
+## Upgrading to Volto 19.x.x
+
+(volto-upgrade-guide-19.x.x)=
+
+### Removed support for loading config from project
+
```

Comment:
Although it may appear redundant so soon after the heading, it provides semantic markup. It also serves as a reminder when adding future headings to also add an admonition as needed.

````suggestion
```{versionremoved} Volto 19
```
````